openshift/template.yaml

```diff
@@ -0,0 +1,173 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: f8a-analytics-ingestion
+metadata:
+  name: f8a-analytics-ingestion
+  annotations:
+    description: f8a-analytics-ingestion
+objects:
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      service: f8a-analytics-ingestion
+    name: f8a-analytics-ingestion
+  spec:
+    replicas: "${{REPLICAS}}"
+    selector:
+      service: f8a-analytics-ingestion
+    strategy:
+      type: Recreate
```

Comment:
This doesn't seem right. I think we should use `Rolling` type. And thus `rollingParams` on the line below.